src/pages/RepoPage/CoverageTab/subroute/FileExplorer/CodeTreeTable/CodeTreeTable.tsx

```diff
@@ -207,6 +195,8 @@ function CodeTreeTable() {
           isMissingHeadReport={isMissingHeadReport}
           hasFlagsSelected={hasFlagsSelected}
           hasComponentsSelected={hasComponentsSelected}
+          isMissingCoverage={pathContentsType === 'MissingCoverage'}
```

Comment:
Do we have an Enum for these pathContentTypes?

Reply:
No but we probably should as it's used multiple areas in the code, i'll draft that up in another PR